### DISCUSSION TITLE
Clarifications around deploying NextJS

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#18546] [Help]
Hi,

I have a question about API Routes, Server Side Rendering and generally anything that involves deploying a NextJS App

What features of next can I use if I deploy the app to another cloud provider ?
Say I want to use API Routes and Server-side rendering: is that possible when deploying to App Engine or is that already considered a custom server, thus loosing Automatic Static Deployment and serverless functions ?

Basically I'd like to understand the implications in terms of deployment for features such as serverless functions and automatic static 

Thanks !
